guides/wide-ep-lws/manifests/modelserver/base/prefill.yaml

```diff
@@ -54,10 +54,6 @@ spec:
               # --enable-dbo: Dual batch overlap (DBO) overlaps compute with collective communication.
               # --enable-eplb:  Expert-parallel load balancing reduces EP load imbalance by replicating heavily-used experts Performance-memory tradeoff: on DeepSeekV3 eplb uses an extra 2GB per redundant expert per GPU Divisibility constraint: num_routed_experts (256 for DSv3) + num_redundant_experts must be divisible by the number of GPUs.
 
-              # Create an environment variable for the vllm venv path so we can
-              # override it for the GKE image as a temporary workaround.
-              # TODO: Once we consolidate the images, we can just hard-code it.
-              source ${VENV_PATH}/bin/activate
```

Comment:
This can be changed to `source /opt/vllm/bin/activate`, and also remove the VENV_PATH override in the coreweave patch.